tests/unit/organizations/test_models.py

```diff
@@ -183,14 +183,14 @@ def test_acl(self, db_session):
             key=lambda x: x[1],
         )
 
-    def test_record_event(self, db_request):
+    def test_record_event_with_geoip(self, db_request):
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏for a future refactor, I'm thinking these test cases should be split out to events/test_models.py, but I wanted to defer that for now, since there's some complexity in creating a subclassed fakeish db model